.github/workflows/tests.yml

```
@@ -56,6 +56,16 @@ jobs:
           unzip -o dcm2niix_lnx.zip
           sudo install dcm2nii* /usr/bin/
 
+      - name: SpinalCord Toolbox installation
+        run: |
+          bash
+          cd ../..
+          git clone --depth 1 https://github.com/neuropoly/spinalcordtoolbox
```

You can pick a specific release like this:

```suggestion
          git clone --depth 1 -b 5.0.1 https://github.com/neuropoly/spinalcordtoolbox
```

5.0.1 [is a tag](https://github.com/neuropoly/spinalcordtoolbox/releases/tag/5.0.1) but this works with branches too!

---

5.0.1 doesn't have those changes in the installer to allow -y to request a non-interactive install, does it?

---

I'm sorry you need to do this by the way. I want you be able to just include SCT in your setup.py, but we're [not there yet](https://github.com/neuropoly/spinalcordtoolbox/issues/1526).

---

No, it doesn't, of course. You're right.